src/components/Dashboard/FilesList.tsx

```diff
@@ -1,10 +1,12 @@
-import React from 'react';
-import Link from 'next/link';
 import dayjs from 'dayjs';
 import relativeTime from 'dayjs/plugin/relativeTime';
-dayjs.extend(relativeTime);
-
 import firebase from 'firebase/app';
+import Link from 'next/link';
+dayjs.extend(relativeTime);
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import Firepad from '../../scripts/firepad';
```

Comment:
It seems that firebase isn't initialized here? @thecodingwizard 

<img width="976" alt="Screen Shot 2023-06-14 at 1 53 46 PM" src="https://github.com/cpinitiative/ide/assets/21228024/69b612c6-ac44-43f8-9ac2-bc9d5a706e25">

Reply:
oops, there were a number of issues caused by my hacky implementation of firepad:

- firepad can only be loaded on the browser side since it can't be server-side rendered, so we have to dynamically import it at runtime
- i deleted the firepad headless code, so I had to re-add it (& remove some unnecessary HTML stuff)

I pushed some changes such that firepad loads properly, but I think the download function needs to be imported or something

Reply:
fixed now